waspc/data/Generator/templates/server/README.md

```diff
@@ -1,5 +1 @@
 `npm start` to run server in development mode (it reloads on changes).
-
```

Comment:
These commands don't exist, so I removed this bit from the server README (even though no one reads that, but why not).